src/handlers/imgix.ts

```diff
@@ -143,3 +143,7 @@ export const maybeSignSource = (source: Source, options?: Record<string, unknown
 export const imageToPng = (url: string, w: number) => {
   return maybeSignUri(url, { fm: 'png', w });
 };
+
+export const getSizedImageUrl = (url: string | undefined, size = 40) => {
+  return url ? maybeSignUri(url, { h: size, w: size }) : undefined;
+};
```

Comment:
meant `getSignedImageUrl`?

Reply:
Meant sized but it also signs. Noticed some of the token icon images coming through the API were huge. Mainly using this to properly size Cloudinary images.